src/hypervisor.cc

```diff
@@ -174,59 +174,59 @@ Hypervisor::Hypervisor(std::string uri, std::string username, std::string passwo
 
 NAN_METHOD(Hypervisor::New)
 {
-  NanScope();
+  Nan::HandleScope scope;
 
-  if (!args.IsConstructCall()) {
+  if (!info.IsConstructCall()) {
     // Invoked as plain function `MyObject(...)`, turn into construct call.
     const int argc = 1;
-    Local<Value> argv[argc] = { args[0] };
-    Local<Function> cons = NanNew<Function>(constructor);
-    NanReturnValue(cons->NewInstance(argc, argv));
+    Local<Value> argv[argc] = { info[0] };
+    Local<Function> cons = Nan::New<Function>(constructor);
+    info.GetReturnValue().Set(cons->NewInstance(argc, argv));
```

Comment:
need to add a `return` in front of this, otherwise execution will continue: https://github.com/nodejs/nan/blob/399b3a54ada39a7cf7a11978ea727eae3686666e/nan.h#L486